benchmark/qwen3-asr-webgpu/results/1.7b-memory/gpumem_watch.ps1

```
@@ -0,0 +1,30 @@
+        $p = Get-Process -Id ([int]$Matches[1]) -ErrorAction SilentlyContinue
+        if ($p -and $p.ProcessName -match 'chrome|headless') { $chrome += $c }
+      }
+    }
+    $sum = ($chrome | Measure-Object CookedValue -Sum).Sum
+    $one = 0
+    foreach ($c in $chrome) { if ($c.CookedValue -gt $one) { $one = $c.CookedValue }; if ($c.CookedValue -gt $peakOne) { $peakOne = $c.CookedValue; $peakOneName = $c.InstanceName } }
+    if ($sum -gt $peakSum) { $peakSum = $sum }
+    $t = [int64]((Get-Date).ToUniversalTime() - [datetime]'1970-01-01').TotalMilliseconds
+    ('{0},{1:F0},{2:F0}' -f $t, ($sum/1MB), ($one/1MB)) | Add-Content -Path $Csv
+    $n++
+  } catch { }
```

PSScriptAnalyzer (1.25.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)